x-pack/test_serverless/functional/test_suites/common/discover_ml_uptime/discover/search_source_alert.ts

```diff
@@ -373,38 +377,38 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
       log.debug('create connector');
       connectorId = await createConnector();
-
-      // TODO: fetching connectors fails server side in Serverless with
-      // "index_not_found_exception: no such index [.kibana_ingest]"
-      if (!(await es.indices.exists({ index: INGEST_SAVED_OBJECT_INDEX }))) {
-        await es.indices.create({ index: INGEST_SAVED_OBJECT_INDEX });
-      }
     });
 
     after(async () => {
-      deleteIndexes([OUTPUT_DATA_VIEW, SOURCE_DATA_VIEW]);
-      const [{ id: adhocRuleId }] = await getAlertsByName(ADHOC_RULE_NAME);
-      await deleteAlerts([adhocRuleId]);
-      await deleteDataView(outputDataViewId);
-      await deleteConnector(connectorId);
+      // clean up what we can in case of failures in some tests which blocked the creation of the following objects
+      try {
+        deleteIndexes([OUTPUT_DATA_VIEW, SOURCE_DATA_VIEW]);
```

Comment:
Nit: Is it possible for `deleteIndexes` to throw? It doesn't look like the function returns the promise or `await`s anything. Maybe we can either make the function return a promise and then `await` it here.